### PR TITLE
NMS-8187: VMware-ManagedEntity does not work with VMware 6

### PR DESCRIPTION
**Pull request by christianpape**
JIRA: http://issues.opennms.org/browse/NMS-8187
Bamboo: http://bamboo.internal.opennms.com:8085/browse/OPENNMS-ONMS591

### CODE REVIEW

**Comment by christianpape** [comment]
@indigo423 Does anyone tested this yet? Bamboo is green. So if someone can confirm that this is working with vSphere 6 we can merge it. 18 is about to be released...

**Comment by mvrueden** [comment]
@christianpape it probably helps if you remove the "work in progress" flag.

**Comment by mvrueden** [comment]
FYI: @indigo423 and @mfuhrmann are going to test this issue.

File: integrations/opennms-vmware/src/test/java/org/opennms/netmgt/collectd/vmware/VmwareViJavaAccessTest.java [hunk 1]
```diff
@@ -120,7 +162,12 @@ protected void relax() {
         mockServiceInstance = createMock(ServiceInstance.class);
 
         // setup ServerConnection
-        mockServerConnection = new ServerConnection(new URL("https://hostname/sdk"), new VimPortType(new WSClient("https://hostname/sdk")), mockServiceInstance);
+        mockServerConnection = new ServerConnection(new URL("https://hostname/sdk"), new VimPortType(new WSClient("https://hostname/sdk") {
+            @Override
+            protected SSLSocketFactory getTrustAllSocketFactory(boolean ignoreCert) throws RemoteException {
```
Comment (indigo423) [inline integrations/opennms-vmware/src/test/java/org/opennms/netmgt/collectd/vmware/VmwareViJavaAccessTest.java]: I would recommend to not use ignoreCert here.

Comment (mvrueden) [inline integrations/opennms-vmware/src/test/java/org/opennms/netmgt/collectd/vmware/VmwareViJavaAccessTest.java]: For the test setup it does not really matter, as the connection is mocked anway.

**Comment by mvrueden** [comment]
@christianpape, what is the status of this issue?

**Comment by mfuhrmann** [comment]
I should test this fix in my environment. But I didn't do it yet. Shame on me... I will try to test it asap.

**Comment by mfuhrmann** [comment]
The error I described here http://issues.opennms.org/browse/NMS-8187 is gone since I'm using Christians patch. The VMware-ManagedEntity service stays up!
So I guess, you can merge it.

**Comment by mvrueden** [comment]
This is good news. I think we should merge this to `foundation-2016` to propagate the changes upwarts correctly. I talked with @christianpape and this should not be a problem. 🔨

**Pull request closed without merging**